### DISCUSSION TITLE
Github code audit symbol or mark to guarantee that the application does not implement malware

### DISCUSSION THREAD

**Discussion by Roger204** [community/community#38763] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

First of all I would like to thank you for Github experience, I really love it. 

I open this discussion to suggest a new functionality that could be implemented in github. I the latest weeks I've heard many times that some security programs written by users, implement some malware rather than just offering an specific functionality (there are some open discussions triggering this topic in the Github community).

I just suggest that Github could have a team of security code auditors which (in the repositories that the users request for it) could analyze the code to "guarantee" that there is no malware. Then Github could reward the user by adding a mark or symbol, indicating that the code was analyzed and no malware was found. To support the cost, Github could even charge the cost (e.g. per number of lines of code) for this malware "mark".

**Comment by ossanna16** [comment]
Hi there @Roger204 and welcome to our community! Thank you for the feedback!

**Comment by jfullstackdev** [comment]
from what I'm seeing, even the Support has a hard time triaging requests that they are receiving

so yes, it can be but they simply can't
